inverse_covariance/model_average.py

```diff
@@ -309,7 +314,7 @@ def __init__(self, estimator=None, n_trials=100, subsample=0.3,
 
         # default to QuicGraphLasso
         if self.estimator is None:
-            self.estimator = QuicGraphLasso()
+            self.estimator = QuicGraphLasso(init_method=init_method)
```

Comment:
should be `init_method=self.init_method` (doesnt actually matter but, like to use self once serialized into the class.